scripts/config_template.yaml

```diff
@@ -1,15 +1,19 @@
 # The data sets to load and how to interpret them
 data:
   data_set_1:
-    group_cols:
-      geography: region
-    rollout: 2023-09-01
+    rollout: [2023-09-01, 2024-09-01]
     filters:
       geography_type: nation
       domain_type: age
       domain: 18+ years
       indicator_type: 4-level vaccination and intent
       indicator: received a vaccination
+      time_type: week
+    keep:
+      geography: geography
+      estimate: estimate
+      time_end: date
```

Comment:
I think it's actually good to keep "time_end" as the name of the column throughout the code. Because one might have a model that takes in the start and end dates? Or, at least, it removes any confusion about what the "date" is

Reply:
That's fine. Especially because `keep` doesn't have to be a dictionary, but just a list of the columns to keep, and it can be a standalone config entry instead of recurring under each data set (like the grouping factors - see above discussion). I'll just have to change some names in the data classes.

Reply:
or we could have `date_column` be a variable somewhere

but then maybe that needs to be at the top level of the code? which is kind of annoying. copy-paste shouldn't be too hard for now